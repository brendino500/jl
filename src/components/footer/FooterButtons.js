import { Button, Typography } from '@material-ui/core'
import useStyles from '../../styles/footerStyles/FooterButtonsStyles'

const buttonText = [
  { label: 'Cookie Policy', url: '' },
  { label: 'Terms of Use', url: '' },
  { label: 'Privacy Policy', url: '' },
  { label: 'Support', url: '' },
  { label: 'Material Usage', url: '' },
]

export default function FooterButtons() {
  const classes = useStyles()

  const getBulletPoint = (index) => {
    if (index < buttonText.length - 1) {
      return <Typography>â€¢</Typography>
    }
  }

  return (
    <div className={classes.root}>
      {buttonText.map((text, index) => (
        <div className={classes.buttonFormat} key={index}>
          <Button className={classes.button} href={text.url}>
            {text.label}
          </Button>
          {getBulletPoint(index)}
        </div>
      ))}
    </div>
  )
}
