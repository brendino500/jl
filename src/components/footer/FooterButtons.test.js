import renderer from 'react-test-renderer'
import FooterButtons from './FooterButtons'

it('FooterButtons matches snapshot', () => {
  const tree = renderer.create(<FooterButtons />).toJSON()
  expect(tree).toMatchSnapshot()
})
