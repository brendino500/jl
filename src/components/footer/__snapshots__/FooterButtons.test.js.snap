// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterButtons matches snapshot 1`] = `
<div
  className="makeStyles-root-1"
>
  <div
    className="makeStyles-buttonFormat-3"
  >
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-2"
      disabled={false}
      href=""
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Cookie Policy
      </span>
    </button>
    <p
      className="MuiTypography-root MuiTypography-body1"
    >
      •
    </p>
  </div>
  <div
    className="makeStyles-buttonFormat-3"
  >
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-2"
      disabled={false}
      href=""
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Terms of Use
      </span>
    </button>
    <p
      className="MuiTypography-root MuiTypography-body1"
    >
      •
    </p>
  </div>
  <div
    className="makeStyles-buttonFormat-3"
  >
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-2"
      disabled={false}
      href=""
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Privacy Policy
      </span>
    </button>
    <p
      className="MuiTypography-root MuiTypography-body1"
    >
      •
    </p>
  </div>
  <div
    className="makeStyles-buttonFormat-3"
  >
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-2"
      disabled={false}
      href=""
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Support
      </span>
    </button>
    <p
      className="MuiTypography-root MuiTypography-body1"
    >
      •
    </p>
  </div>
  <div
    className="makeStyles-buttonFormat-3"
  >
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-2"
      disabled={false}
      href=""
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Material Usage
      </span>
    </button>
  </div>
</div>
`;
